protocol/protobuf/communities.proto

```diff
@@ -23,6 +23,7 @@ message CommunityMember {
     ROLE_TOKEN_MASTER = 5;
   }
   repeated Roles roles = 1;
+  // deprecated, do not put accounts here
   repeated RevealedAccount revealed_accounts = 2;
```

Comment:
```suggestion
  repeated RevealedAccount revealed_accounts = 2 [deprecated=true];
```
Then regenerate protobuf, it will add deprecated comment to golang generated code.